README.md

```diff
@@ -1,2 +1,46 @@
 # kms-json
-Encrypting and decrypting JSON file using AWS Key Management Service (KMS) master key
+Module for encrypting and decrypting JSON objects using AWS Key Management Service (KMS) customer master keys.
+
+## Usage
+
+* Install the package: `npm install kms-json`
+* Require and instantiate `kms-json`:
+  ```javascript
+  const KmsJson = require('kms-json');
+  const kmsJson = new KmsJson({
+      awsKmsSettings: {
+        accessKeyId: 'AKIAIOSFODNN7EXAMPLE',
+        secretAccessKey: 'wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY',
+        region: 'us-east-1',
+        apiVersion: '2014-11-01'
+      },
+      keyId: 'arn:aws:kms:us-east-1:123456789123:key/a7c08fe1-b767-4883-8c94-85726579d57c'
+  });
+  ```
+
+Supported options:
+
+Name                | Type         | Description
```

Comment:
hm... they are just above it, and some of them are too long to fit in the table nicely. Do you think it would add clarity?